client/components/marketing-survey/cancel-purchase-form/stepComponents/test/business-at-step.jsx

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import React from 'react';
+import { shallow } from 'enzyme';
+import { stub } from 'sinon';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { BusinessATStep }	from '../business-at-step';
+
+describe( 'BusinessATStep', function() {
```

Comment:
mocha generally recommends against using arrow functions - `this` is set to something that you may need to use.